circle.yml

```diff
@@ -249,6 +247,24 @@ jobs:
           paths:
             - ~/build/corpus
 
+  spectest:
```

Comment:
I don't see benefits of running this as a separate container. Just add it to "sanitizers" or any other.

Reply:
I like the immediate feedback on the github ci list.

Reply:
I don't think we can afford this for every job. Is this one going to be a special one? You also will not notice if there is sanitizer warnings caused by executing a spectest. Because it runs instantly, I would add it there too.

Reply:
So what are you exactly saying/proposing?

Reply:
At least add it to "sanitizers" too.